acquire/acquire.py

```diff
@@ -1971,6 +1990,7 @@ def upload_files(paths: list[Path], upload_plugin: UploaderPlugin, no_proxy: boo
             History,
             Misc,
             NTDS,
+            ActiveDirectory,
```

Comment:
Can you add it to the default profile too?

Reply:
Yes